### DISCUSSION TITLE
Timeline for Version 2 of Riverpod

### DISCUSSION THREAD

**Discussion by adplant** [rrousselGit/riverpod#1583] [General]
I was just wondering when it was likely that Riverpod 2.x becomes stable. Any info?

**Comment by TimWhiting** [comment]
Initially it was planned for end of August, but I think I saw on Twitter or Github that Remi is planning on September now. I recommend tuning into https://fluttervikings.com/riverpod-20 next week for more info.

**Comment by scopendo** [comment]
Here's Remi's demonstration of Riverpod 2.0 at Flutter Vikings: https://youtu.be/C2Zp731g8Es?t=12763

And a tweet earlier today: https://twitter.com/remi_rousselet/status/1565702396596174849?s=20&t=ib-TjBehQ7RySP_sEjgg9Q
